patch/increase-maximum-gpio-number-for-x86-64.patch

```diff
@@ -0,0 +1,36 @@
+Backport from Linux 5.15
+
+commit d7109fe3a0991a0f7b4ac099b78c908e3b619787
```

Comment:
Please do not use the output of `git show` but create the patch with `git format-patch -1 d7109fe3a0991a0f7b4ac099b78c908e3b619787`, so it can be easily applied with `git am`.